pkg/tool/helm.go

```diff
@@ -27,9 +29,11 @@ func (h Helm) InstallWithValues(chart string, valuesFile string, namespace strin
 		values = []string{"--values", valuesFile}
 	}
 
-	if _, err := h.RunProcessAndCaptureOutput("helm", "install", release, chart, "--namespace", namespace,
-		"--wait", values, h.extraArgs); err != nil {
-		return err
+    execArgs := []string{"install", release, chart, "--namespace", namespace, "--wait"}
+    execArgs = append(execArgs, values...)
+    execArgs = append(execArgs, h.extraArgs...)
+	if _, err := h.RunProcessAndCaptureOutput("helm", execArgs); err != nil {
+        return fmt.Errorf("error installing release: %s -> %w", execArgs, err)
```

Comment:
same question as above.

Reply:
I believe this has been changed in main.